workspaces/kiali/plugins/kiali-backend/src/clients/fetch.test.ts

```diff
@@ -0,0 +1,229 @@
+          strategy: AuthStrategy.token,
+          sessionInfo: {},
+        });
+
+        expect(result.verify).toBeFalsy();
+        expect(result.category).toBe(ValidationCategory.configuration);
+        expect(result.message).toBeDefined();
+        expect(result.message).toStrictEqual(
+          "Attribute 'serviceAccountToken' is not in the backstage configuration",
+        );
+        expect(result.missingAttributes).toBeDefined();
+        expect(result.missingAttributes[0]).toStrictEqual(
+          'serviceAccountToken',
+        );
+      });
+
+      it('should get a true when `serviceAccountToken` is set', async () => {
+        const kialiFetch = new KialiFetcher(
+          {
+            url: 'https://localhost:4000',
+            serviceAccountToken: '<token>',
+            urlExternal: 'https://localhost:4000',
+          },
+          logger,
+        );
+        const result = (kialiFetch as any).validateConfiguration({
+          strategy: AuthStrategy.token,
+          sessionInfo: {},
+        });
+
+        expect(result.verify).toBeTruthy();
+        expect(result.category).toBe(ValidationCategory.unknown);
+        expect(result.message).toBeUndefined();
+        expect(result.missingAttributes).toBeUndefined();
+        expect(result.helper).toBeUndefined();
+      });
+    });
+
+    describe(`Not ${AuthStrategy.openid} strategy supported`, () => {
+      it('should get a true', async () => {
+        const kialiFetch = new KialiFetcher(
+          {
+            url: 'https://localhost:4000',
+            urlExternal: 'https://localhost:4000',
+          },
+          logger,
+        );
+        const result = (kialiFetch as any).validateConfiguration({
+          strategy: AuthStrategy.openid,
+          sessionInfo: {},
+        });
+
+        expect(result.verify).toBeFalsy();
+        expect(result.category).toBe(ValidationCategory.configuration);
+        expect(result.message).toBeDefined();
+        expect(result.message).toStrictEqual(
+          `Strategy ${AuthStrategy.openid} is not supported in Kiali backstage plugin yet`,
+        );
+      });
+    });
+  });
+
+  describe('Read Ca files', () => {
+    describe('bufferFromFileOrString', () => {
+      it('No file or data passed', () => {
+        const kialiFetch = new KialiFetcher(
+          {
+            url: 'https://localhost:4000',
+            urlExternal: 'https://localhost:4000',
+          },
+          logger,
+        );
+        const result = (kialiFetch as any).bufferFromFileOrString();
+
+        expect(result).toBeNull();
+      });
+
+      it('Read from file', () => {
+        const kialiFetch = new KialiFetcher(
+          {
+            url: 'https://localhost:4000',
+            urlExternal: 'https://localhost:4000',
+          },
+          logger,
+        );
+        const result = (kialiFetch as any).bufferFromFileOrString(
+          './__fixtures__/ca_example.pem',
```

Comment:
If you change the file path to  
`${process.cwd()}/plugins/kiali-backend/__fixtures__/ca_example.pem`; 
the tests should pass